src/util/records.py

```diff
@@ -42,7 +41,7 @@
     table_exists,
     target_of,
 )
-from .report import add_to_migration_reports
+from .views.records import *
```

Comment:
```suggestion
from .views.records import add_view, edit_view, remove_view
```

Reply:
Doesn't this somewhat defeat the * import + `__all__` purpose though?
The alternative is that I add the noqa to every single line where the imported names are used, which is kinda spammy.

Reply:
> Doesn't this somewhat defeat the * import + `__all__` purpose though?

Yes, `__all__` is not strictly needed then. But we can keep it.